addons/website_slides/models/slide_slide.py

```diff
@@ -189,7 +190,7 @@ class Slide(models.Model):
     embedcount_ids = fields.One2many('slide.embed', 'slide_id', string="Embed Count")
     slide_views = fields.Integer('# of Website Views', store=True, compute="_compute_slide_views")
     public_views = fields.Integer('# of Public Views', copy=False)
-    total_views = fields.Integer("Views", default="0", compute='_compute_total', store=True)
+    total_views = fields.Integer("# of Views in Total", default="0", compute='_compute_total', store=True)
```

Comment:
"in Total" sound a bit weird, is it in the spec ?

Reply:
I added *in Total* in order to really differentiate them from the website and public views. 
The spec stated that they needed to be grouped so I wanted it to be easy to differentiate the different types of *Views*.

Reply:
Maybe `# Total Views` instead ?  (I do not really like `# of Views in Total`, its too long) What do you think ?

Reply:
Yeah, it's better this way, I am modifying it now.